### PR TITLE
[Ready] Ansible tweaks for Docker SDK

### PR DESCRIPTION
**Pull request by psuter**
@rabbah we can pile up the tweaks here.

### CODE REVIEW

**Comment by rabbah** [comment]
@domdom82 @lehoanganh the tar command used to build the docker black box uses `--transform` which is supported by GNU `tar` but not `bsdtar`. The latter appears to be common to mac users. We need to change the archive approach.

**Comment by lehoanganh** [comment]
@psuter welcome to ansible team

**Comment by rabbah** [comment]
Fixed tar issue and added some other goodies in https://github.com/psuter/openwhisk/pull/1.

**Comment by psuter** [comment]
The goodies made it here.

**Comment by lehoanganh** [comment]
@rabbah @psuter looks good to me.

File: ansible/roles/sdk/tasks/deploy.yml [hunk 1]
```diff
@@ -9,25 +9,26 @@
...
+
 - name: build blackbox container artifact
-  local_action: shell "tar" "--exclude=build.gradle" "--transform" "s/^docker/dockerSkeleton/" "-czf" "{{tmpdir.stdout}}/blackbox-0.1.0.tar.gz" "docker" chdir="{{ openwhisk_home }}/sdk/"
+  local_action: shell "tar" "--exclude=build.gradle" "-czf" "{{ tmpdir.stdout }}/blackbox-0.1.0.tar.gz" "dockerSkeleton" chdir="{{ tmpdir.stdout }}"
 
 - name: copy blackbox container artifact to nginx
-  copy:
-    src: "{{tmpdir.stdout}}/blackbox-0.1.0.tar.gz"
-    dest: "{{ nginx_conf_dir }}"
+  local_action: copy src="{{ tmpdir.stdout }}/blackbox-0.1.0.tar.gz" dest="{{ nginx_conf_dir }}"
```
Comment (lehoanganh) [inline ansible/roles/sdk/tasks/deploy.yml]: would this work in a distributed mode?

Comment (rabbah) [inline ansible/roles/sdk/tasks/deploy.yml]: no since it's `local_action` - good catch, will change it to a copy task as before.

**Comment by psuter** [comment]
Now includes @rabbah's change per @lehoanganh's suggestion.

**Comment by rabbah** [comment]
PPG 656.

**Comment by lehoanganh** [comment]
LGTM

**Comment by markusthoemmes** [comment]
merged as 35eca25f3e2f09ff71ac148001dee0ca7af2ae6d 3348d8be2cb277f63f5c174699e5fa2c884c63fd cbed19c168fb1f55e3b3ccdc4eb7d09435705ba1 adbe4529cd03a5b9b8bb09f00e79956b7fa2e5a7

**Pull request closed without merging**